src/plugins/ovp/components/video-screen/DetailsTab.js

```diff
@@ -6,6 +6,8 @@ import AdvertisingFields from '@triniti/cms/plugins/common/components/advertisin
 import ImagePickerField from '@triniti/cms/plugins/dam/components/image-picker-field/index.js';
 import TaggableFields from '@triniti/cms/plugins/common/components/taggable-fields/index.js';
 import PicklistField from '@triniti/cms/plugins/sys/components/picklist-field/index.js';
+import Medialive from '@triniti/cms/plugins/ovp/components/video-screen/Medialive.js';
+import HighlightsCard from '@triniti/cms/plugins/ovp/components/video-screen/HighlightsCard.js';
```

Comment:
@rsumilang highlights is tmz specific.  would need to have a custom detailstabs in tmz and include those fields there however you like.